### PR TITLE
Constants for features

### PR DESCRIPTION
**Pull request by annando**
We now use constants for the features. No functionality is added.

### CODE REVIEW

File: src/Content/GroupManager.php [hunk 1]
```diff
@@ -172,7 +172,7 @@ public static function widget(int $uid)
 	 */
 	public static function profileAdvanced($uid)
 	{
-		$profile = intval(Feature::isEnabled($uid, 'forumlist_profile'));
+		$profile = Feature::isEnabled($uid, Feature::GROUPS);
 		if (!$profile) {
```
Comment (MrPetovan) [inline src/Content/GroupManager.php]: While you're at it.
```suggestion
		if (!Feature::isEnabled($uid, Feature::GROUPS)) {
```

File: src/Content/Widget.php [hunk 2]
```diff
@@ -598,4 +598,9 @@ public static function channels(string $base, string $channelname, int $uid): st
 			$channelname
 		);
 	}
+
+	public static function visibleForUser(int $uid, string $channel): bool
```
Comment (MrPetovan) [inline src/Content/Widget.php]: This doesn't look relevant to this PR.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Content/Widget.php (hunk 2 above)
Comment (annando) [reply src/Content/Widget.php]: Yeah, a leftover. I thought I had removed it.

File: src/Content/GroupManager.php (hunk 1 above)
Comment (annando) [reply src/Content/GroupManager.php]: Changed

**Review by MrPetovan — APPROVED** [review]
